test/TestProjects/Spread-Cadl/Spread-Cadl.cadl

```diff
@@ -34,8 +34,21 @@ model Thing {
   age: int32;
 }
 
+alias ThingAlias = {
+  @doc("name of the Thing")
+  name: string;
+  @doc("age of the Thing")
+  age: int32;
+};
```

Comment:
Could we create an alias that has a mixture of @path @header and body parameters?
For instance, add another property with `@path` decorator to show if that part could work properly

Reply:
add a new test with multiple targets.